blueprints/model/index.js

```diff
@@ -0,0 +1,22 @@
+var _ = require('lodash');
```

Comment:
Sorry about the additional dependency, but I think `lodash` is worth having around